src/test/js/multibranch/multibranchOpening.js

```diff
@@ -0,0 +1,140 @@
+const jobName = 'withGitFlow';
+const path = require("path");
+const pathToRepo = path.resolve('./target/test2-project-folder');
+const soureRep = './src/test/resources/multibranch_2';
+const git = require("../../../main/js/api/git");
+
+
+module.exports = {
+
+
+    // ** creating a git repo */
+    before: function (browser, done) {
+          // we creating a git repo in target based on the src repo (see above)
+          git.createRepo(soureRep, pathToRepo)
+              .then(function () {
+                  git.createBranch('feature/1', pathToRepo)
+                      .then(done);
+              });
+    },
+    
+    /**
+     * Make sure we can open the master branch results screen from activity
+     */
+    'open master branch from activity': function (browser) {
```

Comment:
I would prefer to do something like:

```
var cases = [{ regexp: 'tr[id^="master"]', name: 'masterActivityMB' }, { regexp: 'tr[id^="master"]', name: 'masterBranchesMB' }, '...'];
        async.mapSeries(cases, function (case, callback) {
             const multibranchCreate = browser.page.multibranchCreate().navigate();      
//the following is why I like ES6 const { name: jobname, regexp} = case; instead of the following
             const jobname = case.name;
             const regexp = case.regexp;
￼             multibranchCreate.createBranch(jobname, pathToRepo);
             const blueActivityPage = browser.page.bluePipelineActivity().forJob(jobName, 'jenkins'); 
￼             blueActivityPage.waitForElementVisible(regexp);
￼             blueActivityPage.click(regexp);
             blueActivityPage.isMultiBranchKosher(createCallbackWrapper(callback));
        });

// in blueActivityPage
isMultiBranchKosher: function (callback) {
         const blueActivityPage = this;
          //check results look kosher:
￼          blueActivityPage.waitForElementVisible('.progress-spinner.running');                           
￼          blueActivityPage.waitForElementVisible('.header.running')
￼          
￼          blueActivityPage.waitForElementVisible('.pipeline-node-selected');                  
￼          blueActivityPage.waitForElementVisible('.download-log-button');                  
￼          blueActivityPage.waitForElementVisible('.pipeline-selection-highlight');                    
￼          blueActivityPage.waitForElementVisible('.pipeline-connector');     
￼          blueActivityPage.waitForElementVisible('.pipeline-node-hittarget');     
￼          blueActivityPage.waitForElementVisible('.success', callback); 
}
```